misc/module_config.json

```diff
@@ -3,6 +3,9 @@
     "ipaddr": {
       "timeout_sec": 300
     },
+    "ipchange": {
```

Comment:
I don't know how to comment on it -- but there's an extra flie "Blank Diagram.png"?